AndroidManifest.xml

```diff
@@ -125,6 +125,12 @@
             android:exported="false"
             android:label="@string/search_users_and_groups_hint" />
 
+        <provider
+            android:name=".ui.adapter.DiskLruImageCacheFileProvider"
```

Comment:
This is for sending cached images to other apps.
Technically only a content:// is sent which points to this provider, as far as I remember.